soh/include/z64save.h

```diff
@@ -290,6 +290,7 @@ typedef struct {
     /*        */ SohStats sohStats;
     /*        */ FaroresWindData backupFW;
     /*        */ RandomizerCheckTrackerData checkTrackerData[RC_MAX];
+    /*        */ u8 maskMemory;
```

Comment:
Going forward I would like all soh additions to save context to be added to sohStats or another struct prefixed with soh.

Reply:
I also think it'd be better to move all soh related data to a substruct so it ends up looking like this:
```c
typedef struct {
    RandoSetting randoSettings[300];
    ItemLocationRando itemLocations[RC_MAX];
    HintLocationRando hintLocations[50];
    EntranceOverride entranceOverrides[ENTRANCE_OVERRIDES_MAX_COUNT];
    char childAltarText[250];
    char adultAltarText[750];
    RandomizerCheck rewardCheck[9];
    char ganonHintText[300];
    char gregHintText[250];
    char ganonText[250];
    char dampeText[150];
    char sheikText[200];
    char sariaText[150];
    char warpMinuetText[100];
    char warpBoleroText[100];
    char warpSerenadeText[100];
    char warpRequiemText[100];
    char warpNocturneText[100];
    char warpPreludeText[100];
    RandomizerCheck masterSwordHintCheck;
    RandomizerCheck lightArrowHintCheck;
    RandomizerCheck sariaCheck;
    RandomizerCheck gregCheck;
    RandomizerCheck dampeCheck;
    char inputSeed[1024];
    u32 finalSeed;
    u8 seedIcons[5];
    u16 randomizerInf[10];
    u8 mqDungeonCount;
    u16 adultTradeItems;
    u8 triforcePiecesCollected;
} SohRandomizerData;

typedef struct {
    u16 pendingSale;
    u16 pendingSaleMod;
    uint8_t questId;
    uint32_t isBossRushPaused;
    uint8_t bossRushOptions[BOSSRUSH_OPTIONS_AMOUNT];
    u8 pendingIceTrapCount;
    SohStats stats;
    FaroresWindData backupFW;
    RandomizerCheckTrackerData checkTrackerData[RC_MAX];
    u8 maskMemory;
    SohRandomizerData randomizer;
} SohData;

typedef struct {
    /* all of the vanilla data */
    /*        */ SohData soh;
} SaveContext; // size = 0x1428
```

But I think that's out of scope of this pr.